staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,7 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := utilnet.CloneTLSConfig(tc.TLSConfig)
+			tlsConfigCopy := tc.TLSConfig
```

Comment:
it could do this instead:
```
// Copy the testcase config
tlsConfigCopy := tc.TLSConfig.Clone()
// Also call Clone() on tlsConfigCopy, since it mutates serverInitOnce
tlsConfigCopy.Clone()
```